test/config/debug.node.config.toml

```diff
@@ -105,8 +105,7 @@ PrivateKeys = [{Path = "./test/sequencer.keystore", Password = "testonly"}]
 
 [Aggregator]
 Host = "0.0.0.0"
-Port = 50081
-ForkId = 2
+Port = 500
```

Comment:
Why this change? I think it is a typo error

Reply:
Yes, is a typo, only Forkid must be deleted